.github/workflows/windows-unittests.yml

```diff
@@ -34,10 +38,6 @@ jobs:
         run: |
           $ErrorActionPreference = 'Stop';
           # install dependencies
-          # The Codecov Python uploader was deprecated on Feb 1, 2022 and disappeared from PyPi on Apr 12, 2023.
-          # python -m pip install codecov
```

Comment:
I'm not sure we should remove completely this part, since we might want to reenable `codecov` at some point. WDYT?

Reply:
If you still want to remove it from the CI though, you can maybe put these lines in the requirements.txt file instead?

Reply:
Usually I'm not a big fan of commented code (or TODO) in git, as you can retrieve it in history or should deserve a JIRA ticket, that's why I removed it. But if you prefer to let it like that I can adapt

Reply:
Since we recently removed the `codecov` python dependency, to use the new codecov uploader installer, we don't need this anyway, so LGTM !